model/config.go

```diff
@@ -87,6 +87,8 @@ const (
 	SERVICE_SETTINGS_DEFAULT_MAX_LOGIN_ATTEMPTS = 10
 	SERVICE_SETTINGS_DEFAULT_ALLOW_CORS_FROM    = ""
 	SERVICE_SETTINGS_DEFAULT_LISTEN_AND_ADDRESS = ":8065"
+	SERVICE_SETTINGS_DEFAULT_GFYCAT_API_KEY     = "2_KtH_W5"
```

Comment:
we should commit these values?

Reply:
Yeah, after discussing with the Gfycat team, we're basically using these as identification tokens vs. authentication, and they'd like to minimize the friction for users. Customers can always manage their own tokens for increased security if they wish.